docs/_templates/autosummary/class_no_inherited_members.rst

```diff
@@ -0,0 +1,42 @@
+{%+ if referencefile %}
+.. include:: {{ referencefile }}
+{%+ endif -%}
+
+{#
+   This is very similar to the default class template, except this one is used
+   when we don't want to generate any inherited methods.
+-#}
+
+{{ objname | escape | underline }}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+{#-
+   Avoid having autodoc populate the class with the members we're about to
+   summarize to avoid duplication.
+#}
+   :no-members:
+   :show-inheritance:
```

Comment:
It'd be nice if this actually gave a link to the parent, but it doesn't look like it does. My guess is because it's returning `qiskit.circuit.quantumcircuit.QuantumCircuit` instead of `qiskit.circuit.QuantumCircuit`. Not sure if there is a simple way to resolve that though.